src/browser.ts

```diff
@@ -143,13 +157,21 @@ export class Browser {
   wsEndpoint() {
     return this.#celestial.ws.url;
   }
+
+  /**
+   * The browser's websocket ready state
+   */
+  wsReadyState() {
+    return this.#celestial.ws.readyState;
+  }
```

Comment:
Sorry I didn't catch this earlier, but is this used anywhere? I'd love some clarity on this.

Reply:
Yeah mostly for testing that the websocket connections were properly closed in the tests to avoid leaking ops

I think maybe this could be changed to a `isClosed` method (if I'm not mistaken users currently cannot detect if a Browser instance has been closed) since assuming the Browser instantiaton didn't failed, the only other interesting state is to know if it has been previously closed

Reply:
I think making this an `isClosed` method (or even a `closed` getter) would be great. I think it's my last nit on this PR.